content/resource-management/drives.md

```diff
@@ -161,6 +161,23 @@ $ kubectl directpv remove --drives=vdb --nodes=node1
 
 Refer to the [remove command]({{< relref "command-line/remove.md" >}}) for more information.
 
+## Repair Drives
+
+{{< admonition title="Data Loss" type="caution" >}}
+THIS IS DANGEROUS OPERATION WHICH LEADS TO DATA LOSS.
+{{< /admonition >}}
+
+In a rare situation, filesystem on a faulty XFS-formatted drive can be repaired to make it usable. 
```

Comment:
Same thought w.r.t. framing.

Reply:
It is, in fact, `xfs_repair` under the hood.
Updated to that suggestion in both places.